### PR TITLE
Made plasma grindable

### PR DESCRIPTION
**Pull request by SethLafuente**
## About the PR 
Made plasma grindable 
**Screenshots**

image

**Changelog**

:cl: Seth
- add: Added Grindable Plasma

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Why is paper grindable seth

**Comment by mirrorcult** [comment]
yeah the grindable tag should be added to just plasma, adding it to the parent makes paper grindable

**Comment by mirrorcult** [comment]
> 
> 
> > Why is paper grindable seth
> 
> sure I can change that but why shouldnt paper be grindable?

because it has no solution container

**Comment by SweptWasTaken** [comment]
> 
> 
> > Why is paper grindable seth
> 
> sure I can change that but why shouldnt paper be grindable?

A. Because your PR isn't "make paper grindable" it's "make plasma grindable"
B. You added it to the parent. Don't.

**Comment by SethLafuente** [comment]
it works fine bb (:

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
You put it on the wrong sheet...

**Comment by SethLafuente** [comment]
> You put it on the wrong sheet...

I’m pretty sure I putted it on right sheet dude I tested it and it works with plasma I can see the code

**Comment by metalgearsloth** [comment]
bruh 
image

**Comment by SethLafuente** [comment]
> bruh
> image

Yeah that’s the right one I was told by Tim rod i asked him if it was right because it should probably be plasma ore and not plasma sheet but he told me that no plasma sheet is this plasma I tested it and he was right that’s the right one

**Comment by metalgearsloth** [comment]
Screenshot me where he told you this

**Comment by TimrodDX** [comment]
> 
> 
> Screenshot me where he told you this

I did, because I assumed that plasma ore was the mining material. I apologize if this was incorrect, though I admitted it was mostly just a hunch at the time.

**Comment by metalgearsloth** [comment]
Why have you not put this on the base plasma sheet Seth?

File: Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml [hunk 1]
```diff
@@ -103,6 +103,15 @@
...
     count: 1
+  - type: SolutionContainer
+    caps: None
+    contents:
+      reagents:
+      - ReagentId: Plasma
+        Quantity: 10
+  - type: Tag
+    tags:
+    - Grindable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml:114]: You put this on `SheetPlasma1`, which is a special prototype that only has ONE sheet of plasma in it.
This needs to be on the base prototype, `SheetPlasma`, which is used for all plasma sheets.

**Review by Zumorica — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml (hunk 1 above)
Comment (SethLafuente) [reply Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml:114]: O ok

**Comment by SweptWasTaken** [comment]
https://hackmd.io/@ss14/yaml

**Comment by SethLafuente** [comment]
> https://hackmd.io/@ss14/yaml
> 
> It's beyond stupid that sloth has to baby you through the very basics of yaml in your own PR. Please do not make anymore until you've grasped at least the fundamental concepts of yaml.

How was I suppose to know there was a special thing for plasma ore what is it even for?

I even tested it with all the plasma and I grinder well?

**Comment by metalgearsloth** [comment]
All the plasma on saltern at roundstart is probably SheetPlasma1. This does not mean it is the only plasma spawnable, hence its parent that was pointed out to you 3 times.

This has literally NOTHING to do with plasma ore and at no point have I said it does.

Which part of its parent being the actual base SheetPlasma is causing confusion?

**Pull request closed without merging**